lib/Agrammon/Web/Service.pm6

```diff
@@ -112,8 +113,8 @@ class Agrammon::Web::Service {
             }
         }
 
-        use Agrammon::OutputFormatter::Text;
-        my $result = output-as-text($!model, $outputs, 'de', 'LivestockTotal');
+        # TODO: what is last parameter needed for?
+        my $result = output-as-text($!model, $outputs, 'de', 'LivestockTotal', True);
```

Comment:
@jnthn This last parameter (`True`) is required, but seems to be unused in the called method (see chat).

Reply:
Addressed in #188.

Reply:
Will rebase shortly.

Reply:
Rebase done.